src/ontology/genepio-edit.owl

```diff
@@ -4570,7 +4570,7 @@ AnnotationAssertion(obo:GENEPIO_0000001 obo:GENEPIO_0000040 "Environmental Isola
 AnnotationAssertion(obo:GENEPIO_0000004 obo:GENEPIO_0000040 "Nitrate")
 AnnotationAssertion(obo:GENEPIO_0000005 obo:GENEPIO_0000040 "GROUP: MIxS")
 AnnotationAssertion(obo:IAO_0000112 obo:GENEPIO_0000040 "micromole per liter")
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000040 "concentration of nitrate")
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000040 "An environmental datum which describes the concentration of nitrate of a sample."@en)
```

Comment:
So it looks like these definitions are direct quotes of the "MIxS" definitions (listed as term source). But since these are GENEPIO versions it's probably alright to have our own ontological definition and then just reference MIxS - do you agree @ddooley?
We can add a separate issue to go through all these MIxS terms and add a mapping to the MIxS codes (e.g. https://genomicsstandardsconsortium.github.io/mixs/0000426/).

Reply:
Yes given that MIxS isn't an OBO foundry ontology we can improve.  Note that there is a big push though by NMDC to improve the whole suite of MIxS specifications to LinkML standards - and using DataHarmonizer, and this probably involves definition improvements.  More at:  https://github.com/GenomicsStandardsConsortium/mixs6.2_release_candidate